examples/all-clusters-app/nrfconnect/Kconfig

```diff
@@ -33,6 +33,9 @@ choice OPENTHREAD_DEVICE_TYPE
 	default OPENTHREAD_MTD
 endchoice
 
+config CHIP_USE_OPENTHREAD_ENDPOINT
+	default y
```

Comment:
Don't we want to enable it for the whole nrfconnect Thread configuration?

Reply:
Maybe for now, let's test both configurations.

Reply:
Okay, to avoid making this PR too large and complicated to review, I will handle it in the next PR. In this case, let's focus on the new Zephyr System Layer implementation.